base/loading.jl

```diff
@@ -508,7 +508,8 @@ function stale_cachefile(modpath, cachefile)
             return true # cache file was compiled from a different path
         end
         for (f,ftime) in files
-            if mtime(f) != ftime
+            # Issue #13606: compensate for Docker images rounding mtimes
+            if mtime(f) ∉ (ftime, floor(ftime))
```

Comment:
Depends what sort of papers it's coming from, but I often see this kind of notation used for open ranges. A set of discrete points would use curly braces in most papers I read.